en/integrations/plugin-packs/procedures/hardware-devices-barco-cs-restapi.md

```diff
@@ -0,0 +1,151 @@
+---
+id: hardware-devices-barco-cs-restapi
+title: Barco ClickShare Rest API
+---
+
+## Plugin Pack Assets
+
+### Monitored Objects
+
+The Plugin Pack Barco ClickShare collects metrics for:
```

Comment:
When only one Service it could be nice to avoid lists. Example:

The Barco ClickShare Pack collects metrics and status from various Barco hardware devices (CSE 100, CSE-200+, CSE-800, ...).